### ISSUE TITLE
到现在还没运行成功你们的demo,点击加载按钮报错，本机浏览器访问返回。有没有交流群啊

### ISSUE THREAD

**Issue by s3hengwu** [ctripcorp/CRN#21]
![33337](https://user-images.githubusercontent.com/8500025/56329657-0f976400-61b7-11e9-84fa-d08e903f9418.png)

**Comment by s3hengwu** [comment]
![8](https://user-images.githubusercontent.com/8500025/56275903-2ea0e200-6134-11e9-9e23-d12a351712fd.png)

**Comment by s3hengwu** [comment]
本机浏览器访问：http://127.0.0.1:8081/index.bundle?CRNModuleName=CRNDemo&CRNType=1

{"originModulePath":"E:\\reactnative\\crnDemo\\node_modules\\react-native\\Libraries\\react-native\\react-native-implementation.js","targetModuleName":"AccessibilityInfo","message":"Unable to resolve module `AccessibilityInfo` from `E:\\reactnative\\crnDemo\\node_modules\\react-native\\Libraries\\react-native\\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map\n\nThis might be related to https://github.com/facebook/react-native/issues/4968\nTo resolve try the following:\n  1. Clear watchman watches: `watchman watch-del-all`.\n  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.\n  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.\n  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.","errors":[{"description":"Unable to resolve module `AccessibilityInfo` from `E:\\reactnative\\crnDemo\\node_modules\\react-native\\Libraries\\react-native\\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map\n\nThis might be related to https://github.com/facebook/react-native/issues/4968\nTo resolve try the following:\n  1. Clear watchman watches: `watchman watch-del-all`.\n  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.\n  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.\n  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`."}],"name":"Error","stack":"Error: Unable to resolve module `AccessibilityInfo` from `E:\\reactnative\\crnDemo\\node_modules\\react-native\\Libraries\\react-native\\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map\n\nThis might be related to https://github.com/facebook/react-native/issues/4968\nTo resolve try the following:\n  1. Clear watchman watches: `watchman watch-del-all`.\n  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.\n  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.\n  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.\n    at ModuleResolver.resolveDependency (E:\\reactnative\\crnDemo\\node_modules\\metro\\src\\node-haste\\DependencyGraph\\ModuleResolution.js:183:15)\n    at ResolutionRequest.resolveDependency (E:\\reactnative\\crnDemo\\node_modules\\metro\\src\\node-haste\\DependencyGraph\\ResolutionRequest.js:52:18)\n    at DependencyGraph.resolveDependency (E:\\reactnative\\crnDemo\\node_modules\\metro\\src\\node-haste\\DependencyGraph.js:274:16)\n    at Object.resolve (E:\\reactnative\\crnDemo\\node_modules\\metro\\src\\lib\\transformHelpers.js:261:42)\n    at dependencies.map.result (E:\\reactnative\\crnDemo\\node_modules\\metro\\src\\DeltaBundler\\traverseDependencies.js:399:31)\n    at Array.map (<anonymous>)\n    at resolveDependencies (E:\\reactnative\\crnDemo\\node_modules\\metro\\src\\DeltaBundler\\traverseDependencies.js:396:18)\n    at E:\\reactnative\\crnDemo\\node_modules\\metro\\src\\DeltaBundler\\traverseDependencies.js:269:33\n    at Generator.next (<anonymous>)\n    at asyncGeneratorStep (E:\\reactnative\\crnDemo\\node_modules\\metro\\src\\DeltaBundler\\traverseDependencies.js:87:24)"}

**Comment by blackwuxin** [comment]
访问`http://127.0.0.1:8081/index.bundle?CRNModuleName=CRNDemo&CRNType=1&platform=ios` 或者`http://127.0.0.1:8081/index.bundle?CRNModuleName=CRNDemo&CRNType=1&platform=android`

**Comment by s3hengwu** [comment]
访问失败啊
